src/lib.rs

```diff
@@ -970,6 +1062,65 @@ unsafe impl<T: AsBytes> AsBytes for Unalign<T> {
     }
 }
 
+impl<T: Unaligned> Deref for Unalign<T> {
+    type Target = T;
+
+    fn deref(&self) -> &T {
+        // SAFETY: `deref_unchecked`'s safety requirement is that `self` is
+        // aligned to `align_of::<T>()`, which is guaranteed by `T: Unaligned`.
```

Comment:
Suggest: `Unaligned` types are guaranteed to have an alignment of one. All pointers are one-aligned because all addresses are divisible by 1.

Reply:
Good call; done.